conf/deployment/vsphere/upi_1az_rhcos_vsan_1mw.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
is UPI the main deployment method for SNO? @shyRozen can we check with PM?

Reply:
I don't think openshift-install can provision the infrastructure for SNO.
I've just followed the instruction how to install SNO which there are two kinds of install:
I'm using [manual iso install](https://docs.openshift.com/container-platform/4.9/installing/installing_sno/install-sno-installing-sno.html#generating-the-discovery-iso-manually_install-sno-installing-sno-with-the-assisted-installer)
The other way is through assisted installer which is easier but you can use only GA OCP version and for that reason I chose the manual that you can use ci version of OCP [assited installer](https://docs.openshift.com/container-platform/4.9/installing/installing_sno/install-sno-installing-sno.html#install-sno-generating-the-discovery-iso-with-the-assisted-installer_install-sno-installing-sno-with-the-assisted-installer)

Reply:
Checked with Chris and it's only UPI for now. He in one of next version of OCP there will be IPI but it doesn't interest us.